Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN set -xeu && \
         curl ca-certificates ca-certificates-java \
         && \
     mkdir /graalvm && \
-    curl -fsSL "https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz" \
+    curl -fsSL "https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-linux-amd64-${GRAAL_VERSION}.tar.gz" \
         | tar -zxC /graalvm --strip-components 1 && \
     echo 'HOTFIX for missing certificates: https://github.com/oracle/graal/issues/378' && \
     cp /etc/ssl/certs/java/cacerts /graalvm/jre/lib/security/cacerts && \
```

Comment:
Probably workaround not needed anymore But I'm not sure. No bad thing happened if we leave it here.